src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeDateFormatter.java

```diff
@@ -59,7 +60,7 @@ public String getDescription() {
                 "uuuu-M-d", "uuuu-M",
                 "M/uu", "M/uuuu",
                 "MMMM d, uuuu", "MMMM, uuuu",
-                "d.M.uuuu"};
+                "d.M.uuuu", "uuuu.M.d"};
```

Comment:
Mhhh...according to https://en.wikipedia.org/wiki/Date_format_by_country this format is only used by Korea and Mongolia. Do you think it is really worth adding?